src/PasswordField.js

```diff
@@ -126,7 +126,7 @@ class PasswordField extends React.Component {
         <div style={styles.inputContainer}>
           <TextField
             {...other}
-            errorStyle={styles.error}
+            errorStyle={errorStyle || styles.error}
```

Comment:
Why not `{{ ...styles.error, ...errorStyle }}` as with `style`? That way, styles would be merged.